FireRender.Maya.Src/scripts/common.mel

```diff
@@ -207,3 +207,155 @@ global proc checkMattePassAOV()
 	}
 }
 
+// Set dafault values for ramp when there is only one point, sits at the beggining and is black
+// Maya creates exactly that as default value for ramp so this will break if they change it
+// also works if nothing is created
+global proc AERPRCreateDefaultRampColors( string $nodeName, string $colorSrcString, string $posSrcString, string $rampName)
```

Comment:
Is it possible to refactor in order to remove code duplicate in this function and same for Floats?